xt/lib/PageObject/App/Menu.pm

```diff
@@ -76,14 +76,14 @@ my %menu_path_pageobject_map = (
 sub _verify {
     my ($self) = @_;
 
-    my @logged_in_found =
-        $self->find_all('*contains', text => "Logged in as");
-    my @logged_into_found =
-        $self->find_all('*contains', text => "Logged into");
+    my @logged_in_company =
+        $self->find_all("//*[\@id='company_info' and string-length(normalize-space(text())) > 0]");
+    my @logged_in_login =
+        $self->find_all("//*[\@id='login_info' and string-length(normalize-space(text())) > 0]");
```

Comment:
Why do you stop checking the exact text? If there's an important reason to do that, it probably deserves mention in the log message. If there's not, I'd prefer we stick to checking that the screen has *exactly* the desired text.